src/EA.Weee.Web/Views/Shared/_DownloadSubmissionPdfLayout.cshtml

```diff
@@ -0,0 +1,266 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+
+    <title>GOV.UK - The best place to find government services and information</title>
+
+    <style type="text/css">
+
+        .govuk-full-printable-width {
+            width: 100% !important;
+            display: flex;
+        }
+
+            .govuk-full-printable-width span:nth-child(1) {
+                width: 100%;
+            }
+
```

Comment:
@will-hope98  Bit of a pain but would you be able to see if you can move the head and footer into partials that can be re-used across this and the evidence note PDF?

Reply:
Moved to use one partial